### ISSUE TITLE
asgs-brew failed on lonestar5 due to error installing perl modules

### ISSUE THREAD

**Issue by jasonfleming** [jasonfleming/asgs#171]
Hey @wwlwpd I tried installing the latest master asgs onto lonestar5 and everything was going well until well into the perlbrewing process. Here is my command line for asgs-brew.pl and the directory I was in when I executed asgs-brew.pl:

```
login1.ls5(1012)$ perl ./cloud/general/asgs-brew.pl --compiler=intel --machinename=lonestar5 --install-path=$WORK/opt
login1.ls5(1011)$ pwd
/work/00976/jgflemin/lonestar/asgs/jasonfleming/master
```
and here is the error that ended the installation process:

```
Test::More is up to date. (1.302170)
install is up to date. (0.01)
--> Working on Time::Local
Fetching http://www.cpan.org/authors/id/D/DR/DROLSKY/Time-Local-1.28.tar.gz ... OK
Configuring Time-Local-1.28 ... OK
Building and testing Time-Local-1.28 ... OK
Successfully installed Time-Local-1.28 (upgraded from 1.25)
1 distribution installed
install is up to date. (0.01)
Try::Tiny is up to date. (0.30)
Cannot detect source of '/home1/00976/jgflemin/asgs/cloud/general/t/verify-'! at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Parser/IteratorFactory.pm line 256.
	TAP::Parser::IteratorFactory::detect_source(TAP::Parser::IteratorFactory=HASH(0x1b070d0), TAP::Parser::Source=HASH(0x1b25d80)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Parser/IteratorFactory.pm line 211
	TAP::Parser::IteratorFactory::make_iterator(TAP::Parser::IteratorFactory=HASH(0x1b070d0), TAP::Parser::Source=HASH(0x1b25d80)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Parser.pm line 472
	TAP::Parser::_initialize(TAP::Parser=HASH(0x1a0ddb8), HASH(0x18d72e0)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Object.pm line 55
	TAP::Object::new("TAP::Parser", HASH(0x18d72e0)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Object.pm line 130
	TAP::Object::_construct(TAP::Harness=HASH(0x17f7b30), "TAP::Parser", HASH(0x18d72e0)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Harness.pm line 852
	TAP::Harness::make_parser(TAP::Harness=HASH(0x17f7b30), TAP::Parser::Scheduler::Job=HASH(0x1a0d9f8)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Harness.pm line 651
	TAP::Harness::_aggregate_single(TAP::Harness=HASH(0x17f7b30), TAP::Parser::Aggregator=HASH(0x13b2dc8), TAP::Parser::Scheduler=HASH(0x1a0da28)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Harness.pm line 743
	TAP::Harness::aggregate_tests(TAP::Harness=HASH(0x17f7b30), TAP::Parser::Aggregator=HASH(0x13b2dc8), "/home1/00976/jgflemin/asgs/cloud/general/t/verify-perl-modules.t") called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Harness.pm line 558
	TAP::Harness::__ANON__() called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/TAP/Harness.pm line 571
	TAP::Harness::runtests(TAP::Harness=HASH(0x17f7b30), "/home1/00976/jgflemin/asgs/cloud/general/t/verify-perl-modules.t") called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/App/Prove.pm line 548
	App::Prove::_runtests(App::Prove=HASH(0x137bd78), HASH(0x17bfe60), "/home1/00976/jgflemin/asgs/cloud/general/t/verify-perl-modules.t") called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/lib/5.28.2/App/Prove.pm line 506
	App::Prove::run(App::Prove=HASH(0x137bd78)) called at /home1/00976/jgflemin/perl5/perlbrew/perls/perl-5.28.2/bin/prove line 13
post condition for "Step for installing required Perl modules" FAILED, stopping. Please fix and rerun.
```
Maybe still a path variable that is using $HOME instead of the path to my asgs repo?

**Comment by jasonfleming** [comment]
I tried this on Frontera and ran into the same issue. I dug into the asgs-brew.pl script and found an issue with the path supplied to `prove` ... it had `$HOME/asgs...` at the start of the path rather than `./cloud...` I corrected this and made a couple other fixes and took some notes. There are some issues with git on Frontera right now, so I have submitted a support ticket for that. Once that ticket is resolved, I will create a PR with the changes related to the brew and install scripts.

**Comment by wwlwpd** [comment]
I am looking at this now. Nice find. I will make sure it can install from and be installed to the work dir.

**Comment by wwlwpd** [comment]
@jasonfleming - let me know if this fixes this issue for you, https://github.com/jasonfleming/asgs/pull/172

**Comment by jasonfleming** [comment]
Perfect! Closed by #172
